igor/igor-core/src/main/java/com/netflix/spinnaker/igor/history/EchoService.java

```diff
@@ -24,5 +24,5 @@
 /** Posts new build executions to echo */
 public interface EchoService {
   @POST(".")
-  Call<String> postEvent(@Body Event event);
+  Call<Void> postEvent(@Body Event event);
```

Comment:
Why the change in return value?

Reply:
It's because the echo api doesn't return anything: https://github.com/spinnaker/spinnaker/blob/main/echo/echo-web/src/main/java/com/netflix/spinnaker/echo/history/HistoryController.java#L36

Reply:
Fair enough, though it seems like an unrelated change...could it happen in a separate commit/PR, or does the new de/serializer code only work if this is Void?

Reply:
Ehh, let's go for it.